code/style.css

```diff
@@ -2,58 +2,79 @@
   box-sizing: border-box;
 }
 
+@font-face {
+  font-family: wineFont;
+  src: url(./assets/LSLeaves.ttf);
+}
+
+html {
+  height: 100%;
+}
+
 body {
   margin: 0;
   padding: 0;
   font-family: 'Montserrat', sans-serif;
-  background: #0026ff;
+  background-image: url(./assets/vinyard-house.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  overflow: hidden; 
+  height: 100%;
 }
 
 h1 {
-  font-weight: bold;
-  font-size: 44px;
+font-size: 20px;
   line-height: 34px;
-  color: #fff;
+  color: rgb (14, 11, 11);
   text-align: center;
+  letter-spacing: 3px;
+  text-shadow: rgb(255, 255, 255) 0px 0px 40px;
```

Comment:
nice trick with the shadow which creates better contrast for your letters on the background image 🤓